src/cljs/ui/events/continuous_mcc_tree.cljs

```diff
@@ -173,12 +173,14 @@
                                             :mostRecentSamplingDate (time/format date)}}]}))
 
 (defn set-time-scale-multiplier [{:keys [db]} [_ value]]
-  (let [id (get-in db [:new-analysis :continuous-mcc-tree :id])]
+  (let [id (get-in db [:new-analysis :continuous-mcc-tree :id])
+        tsm (let [nval (ui-utils/fully-typed-number value)]
```

Comment:
Personal prefference, but I don't like these shortcut names, I will forget in a month what this stands for.
Just use full name please `timescale-multiplier`, let the goog closure compiler handle making JS bundles unreadable :smile:

Reply:
fair enough, done!